maintenance/importConfig.ts

```diff
@@ -284,11 +300,16 @@ async function parseOZWConfig(): Promise<void> {
 			const products = ensureArray(man.Product);
 			for (const product of products) {
 				if (product.config !== undefined) {
-					await parseOZWProduct(
-						product,
-						manufacturerId,
-						manufacturerName,
-					);
+					if (
+						program.ids === undefined ||
```

Comment:
!program.ids || ...

Reply:
Usually typescript wants strict checks but ok :)

Reply:
well, when program.ids is either an array or undefined, the shorter line will do just fine ;)